common/src/Model/Brush.h

```diff
@@ -184,7 +184,7 @@ namespace TrenchBroom {
             void removeVertices(const vm::bbox3& worldBounds, const std::vector<vm::vec3>& vertexPositions);
 
             bool canSnapVertices(const vm::bbox3& worldBounds, FloatType snapTo);
-            void snapVertices(const vm::bbox3& worldBounds, FloatType snapTo);
+            void snapVertices(const vm::bbox3& worldBounds, FloatType snapTo, bool uvLock = false);
```

Comment:
Why a default parameter?

Reply:
No good reason except to avoid updating the tests, and that `moveVertices`/`moveEdges`/`moveFaces` also do it.